tests/test_core.py

```diff
@@ -1762,7 +1761,6 @@ def test_bigswitch(self):
 
   @no_emterpreter
   def test_biggerswitch(self):
```

Comment:
Actually the "bitcodize inputs" stage takes less than a second with fastcomp and 141s with the wasm backend (!!). So not really sure what the deal is there, I don't know of any weird differences in clang that would cause that.